src/Util/StringUtil.php

```diff
@@ -18,25 +18,25 @@
  */
 final class StringUtil
 {
-    public static function splitLines($input)
+    public static function splitLines(string $input): string
```

Comment:
But our test suite only supports (officially) php 7.0.8 currently. I must change to `array` by now.